gifted_build/src/components/special_olympics.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import "../css/special_olympics.css"
+
+class Special_Olympics extends React.Component<{}, {}> {
+    render () {
+        return (
+            <div id="container">
```

Comment:
rather than using your own written css, this can be taken care of with a css library like bootstrap or bulma. Its up to you if you want to design the css yourself, but you're going need to center this page